src/main/java/teammates/ui/webapi/request/FeedbackSessionStudentSaveRequest.java

```diff
@@ -0,0 +1,23 @@
+package teammates.ui.webapi.request;
+
+/**
+ * The save request of a list of students from feedback session.
+ */
+public class FeedbackSessionStudentSaveRequest extends BasicRequest {
```

Comment:
But this is not a "save" request?

Reply:
is it better to use `remindRequest` or `submitRequest` then?

Reply:
Which one do you think is better?

Reply:
I would go with `remindRequest`.

The submit buttons for both modals states `remind`. Given that `saveRequest` is used for the save button and `createRequest` too. I think `remindRequest` will be more adequate.